sdks/python/apache_beam/ml/inference/tensorrt_inference.py

```diff
@@ -0,0 +1,213 @@
+#
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved. # pylint: disable=line-too-long
```

Comment:
@azhurkevich Does `SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.` have to be on one line? I made some formatting fixes, and added `# pylint: disable=line-too-long` for now to keep it all on one line.

But if it can be broken up into two lines, then it'll look better if we make that change.

Reply:
@yeandy One sec, checking with legal.

Reply:
@yeandy We can do this:
```
SPDX-FileCopyrightText: NVIDIA CORPORATION & AFFILIATES
Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
Or we can do this:
```
SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES.
All rights reserved.
```
Whatever you like more.

Reply:
The header should be the same as other files:

```
# Licensed to the Apache Software Foundation (ASF) under one or more
# contributor license agreements.  See the NOTICE file distributed with
# this work for additional information regarding copyright ownership.
# The ASF licenses this file to You under the Apache License, Version 2.0
# (the "License"); you may not use this file except in compliance with
# the License.  You may obtain a copy of the License at
#
#    http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

Other information about the origin of the code should go into NOTICE

Reply:
This may help: https://www.apache.org/foundation/license-faq.html#provenance

Reply:
Since we are having this discussion, do you have an individual or corporate contribution license agreement on file with ASF?

Reply:
@kennknowles I was added to Google CLA list, whatever that means. Have no idea about what is ASF. 

With regards to header, I've reached out to legal. A person who was looking into my case is OOTO til next Monday, so hopefully he'll come and will resolve this issue. I am not legal, so I'd rather delegate this to him at the same time improving code on my end. Hope it's understandable.

Reply:
@kennknowles "Other information about the origin of the code should go into NOTICE" what does that mean in practice? Can you explain step by step, thanks.

Reply:
The simplest thing is to use the header and nothing else.

The Google CCLA (Corporate Contributor License Agreement) is an agreement between Google and the Apache Software Foundation.